guides/source/active_record_postgresql.md

```diff
@@ -600,6 +601,23 @@ end
 
 By default `:deferrable` is `false` and the constraint is always checked immediately.
 
+Exclusion Constraints
+---------------------
+
+* [exclusion constraints](https://www.postgresql.org/docs/current/ddl-constraints.html#DDL-CONSTRAINTS-EXCLUSION)
+
+```ruby
+# db/migrate/20131220144913_create_products.rb
+create_table :products do |t|
+  t.integer :price, null: false
+  t.daterange :availability_range, null: false
+
+  t.exclusion_constraint "price WITH =, availability_range WITH &&", using: :gist, name: "price_check"
+end
+```
+
+Like foreign keys, exclusion constraints can be deferred by setting `:deferrable` to either `true` or `:immediate`. By default, `:deferrable` is `false` and the constraint is always checked immediately.
```

Comment:
@agrobbin Thank you for writing the document 😸 
The deferrable option is not allowed to pass `true`. [source code](https://github.com/rails/rails/blob/cbc7b59749d582f6529b085ad7facee0a0c0623f/activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb#L678)
The `deferrable: true` option for `foreign_key` will also be deprecated in #47659.

Reply:
Thanks @alpaca-tc! I've updated this to reflect the current reality.